api/accesslog.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "api/base.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+
+message EnvoyAccessLog {
+  // The request method, ie, "GET", "HEAD", "POST"
+  string request_method = 1;
```

Comment:
enum sounds good, BUT, there is an internal proto which uses string for request_method.  Let's discuss offline.